### PR TITLE
3/file content

### LINKED ISSUE #3: Add hooks for getting file content

### PR DESCRIPTION
**Pull request by pyphilia**
close #3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,13 +43,15 @@
...
     "prettier": "2.2.1",
     "pretty-quick": "3.1.0",
-    "react": "link: ./example/node_modules/react",
-    "react-dom": "link: ./example/node_modules/react-dom",
-    "react-scripts": "link: ./example/node_modules/react-scripts",
     "standard-version": "9.1.0",
     "typescript": "4.1.3",
     "wait-on": "5.3.0"
   },
+  "localDependencies": {
```
Comment (juancarlosfarah) [inline package.json:50]: Nice. I didn't know this existed.

File: src/mutations/item.ts [hunk 2]
```diff
@@ -125,7 +126,19 @@ export default (queryClient: QueryClient, queryConfig: QueryClientConfig) => {
...
           },
         }),
-        item: await mutateItem({ id: newItem.id, value: newItem }),
+        item: await (async () => {
+          const itemKey = buildItemKey(newItem.id);
+          await queryClient.cancelQueries(itemKey);
+          const prevValue = queryClient.getQueryData(itemKey) as Record<Item>;
+          queryClient.setQueryData(
+            itemKey,
+            Map({
```
Comment (juancarlosfarah) [inline src/mutations/item.ts:135]: Is there no way of doing this with Immutable? I think you should be able to replace keys.

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Looks good to me! Just one question about updating an Immutable map.

File: src/mutations/item.ts (hunk 2 above)
Comment (pyphilia) [reply src/mutations/item.ts:135]: You're right, `.merge` is exactly what I needed.

**Pull request merged**